src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/ProcessState.cs

```diff
@@ -1,42 +1,43 @@
-using Newtonsoft.Json;
 using System;
 
+using Newtonsoft.Json;
+
 namespace Altinn.Platform.Storage.Interface.Models
 {
     /// <summary>
-    /// Holds the process state of an application instance.
+    /// Represents a status object that holds the process state of an application instance.
     /// The process is defined by the application's process specification BPMN file. 
     /// </summary>
     [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
     public class ProcessState
     {
         /// <summary>
-        /// Indicates the date and time when the process was initialized
+        /// Gets or sets the date and time for when the process was started.
```

Comment:
did we introduce more stylecop rules or are you only changing these of habit ? 
Asking for a friend.. :p

Reply:
Mostly out of habit, but I also like the wording. Microsoft is using the same phrase in SDK documentation. Our texts are used for API documentation so I'm not 100% sure if it fits.